src/helpers/formatters/file.js

```diff
@@ -23,25 +26,13 @@ export const tags = (tags, file) => {
   }
 }
 
-const formatImage = data => {
-  if (data) {
-    const imageFormat = data.data.format
-
-    const processImageByte = byte => {
-      return String.fromCharCode(byte)
-    }
-
-    const imageDecoded = data.data.data.map(
-      processImageByte
-    ).join('')
-
-    const imageBase64 = window.btoa(
-      imageDecoded
-    )
-
-    return [
-      `data:${imageFormat}`,
-      `base64,${imageBase64}`
-    ].join(';')
+/**
+ * Translate music-metadata picture object into a Data URI
+ * @param picture Cover
+ * @returns {string} Data URI
+ */
+function formatImage (picture) {
+  if (picture) {
+    return `data:${picture.format};base64,${picture.data.toString('base64')}`
```

Comment:
@Borewit Please format it like this:
```
return [
  `data:${imageFormat}`,
  `base64,${imageBase64}`
].join(';')

Reply:
You prefer to replace a single semicolon declaration like an array, and then join the elements with a semicolon. I am here mate.

Reply:
Done